lnwallet/channel.go

```diff
@@ -3730,17 +3737,15 @@ func (lc *LightningChannel) validateCommitmentSanity(theirLogCounter,
 		// stuck because locally we will never put another outgoing HTLC
 		// on the channel state. The FeeBuffer should ONLY be enforced
 		// if we locally pay for the commitment transaction.
-		theirBalance, bufferAmt, err = lc.applyCommitFee(
+		theirBalance, bufferAmt, commitFee, err = lc.applyCommitFee(
```

Comment:
nit: while we're touching this, can we also fix the formatting of the closing parenthesis here and above please?

Reply:
fixed.